sdk/keyvault/keyvault-certificates-perf-tests/package.json

```diff
@@ -42,9 +76,8 @@
     "unit-test": "echo skipped",
     "unit-test:browser": "echo skipped",
     "unit-test:node": "echo skipped",
-    "update-snippets": "echo skipped"
+    "update-snippets": "dev-tool run update-snippets"
```

Comment:
do we need update-snippets for perf tests?

Reply:
Just for metrics it would be nice so that I can detect that all projects are snippets complete